rust/foxglove_agent/agent_sdk.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
It feels a little odd for the C deps to live loose in the directory structure alongside Rust files; what do you think about adding them to a dedicated subdirectory?